src/components/layouts/collection-page-layout.tsx

```diff
@@ -142,6 +143,11 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
   const {viewer} = useViewer()
   const viewerId = viewer?.id
   const {data: completedCourses} = useCompletedCourses(viewerId)
+
+  const {data: courseProgress} = trpc.progress.forPlaylist.useQuery({
+    slug: course.slug,
+  })
```

Comment:
this can be used in any component to get current user's progress for given playlist